### PR TITLE
Features/student profile card

### PR DESCRIPTION
**Pull request by lauratsui**
Created student profile card with dummy prop/text. English proficiency will show up as a Select component if status is "screening". Otherwise English Proficiency will show up as text.

### CODE REVIEW

File: frontend/src/components/StudentProfileCard.vue [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+  </div>
+</template>
+
+<script>
+export default {
+  name: "StudentProfileCard",
+  props: {
+    studentInfo: {
+      type: Object,
+      default: function() {
```
Comment (aelbore) [inline frontend/src/components/StudentProfileCard.vue:64]: shortcut function in object
```javascript
{
 default() {
  return { ... }
 }
}
```

**Review by aelbore — APPROVED** [review]
looks good

File: frontend/src/components/StudentProfileCard.vue (hunk 1 above)
Comment (lauratsui) [reply frontend/src/components/StudentProfileCard.vue:64]: ooh neat thanks!

**Review by dzldebby — APPROVED** [review]
looks good!

**Pull request merged**